widget-showcase/tests/unit/all.ts

```diff
@@ -0,0 +1,639 @@
+import * as registerSuite from 'intern!object';
```

Comment:
This should be `app.ts` and the `all.ts` file should just have `import './app';`

Reply:
Done 👍